infrastructure/smart-contracts/scripts/tiny-keys/tinyKeysAirdropDeployer.mjs

```diff
@@ -10,18 +10,34 @@ async function main() {
 
     // get the deployer
     const [deployer] = (await ethers.getSigners());
-    const deployerAddress = await deployer.getAddress();
+    console.log("Deploying contracts with the account:", deployer.address);
   
     // deploy tiny keys airdrop contract
     console.log("Deploying Tiny Keys Airdrop...");
     const TinyKeysAirdrop = await ethers.getContractFactory("TinyKeysAirdrop");
     const tinyKeysAirdrop = await upgrades.deployProxy(TinyKeysAirdrop, [NODE_LICENSE_CONTRACT, REFEREE_CONTRACT, POOL_FACTORY_CONTRACT, KEY_MULTIPLIER], { kind: "transparent", deployer });
     await tinyKeysAirdrop.deploymentTransaction();
     const tinyKeysAirdropAddress = await tinyKeysAirdrop.getAddress();
-    console.log("Tiny Keys Airdrop deployed to:", tinyKeysAirdropAddress);    
+    console.log("Tiny Keys Airdrop deployed to:", tinyKeysAirdropAddress);   
+    
+    console.log("Deploying Referee Calculations Contract...");
+    const RefereeCalculations = await ethers.getContractFactory("RefereeCalculations");
+    const refereeCalculations = await upgrades.deployProxy(RefereeCalculations, [], { kind: "transparent", deployer });
+    await refereeCalculations.deploymentTransaction();
+    const refereeCalculationsAddress = await refereeCalculations.getAddress();
+    console.log("Referee Calculations deployed to:", refereeCalculationsAddress);
 
-    // verify contract
-    await safeVerify({ contract: tinyKeysAirdrop });
```

Comment:
I had high hopes for the safeVerify did it not work compared to running verify here ?

Reply:
I updated it to use safeVerify and it did work. 

Previously there was an issue with the import directory being wrong and the .mjs file did not flag the error. Once that was updated, it worked perfectly.

Reply:
Thats perfect, thanks for making it work!